projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts

```diff
@@ -462,15 +454,11 @@ export class TuiInputCardGroupedComponent
         }
     }
 
-    private updateProperty(propValue: string, propName: 'card' | 'cvc' | 'expire'): void {
-        const {card, expire, cvc} = this.value || STUB;
-        const newValue: TuiCard = {
-            card,
-            expire,
-            cvc,
-        };
+    private updateProperty(value: string, propName: 'card' | 'cvc' | 'expire'): void {
+        const {card = '', expire = '', cvc = ''} = this.value ?? {};
```

Comment:
nit: maybe `||` instead of `??` ? `||` a little faster and transpiles to smaller JavaScript code than "??"

Reply:
fixed